style.css

```diff
@@ -1,4 +1,4 @@
-// неправильный комментарий
+// неправильный комментарий 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/test_test/pullRequest?prid=5457187)